ui-tests/src/test/resources/features/connections/oauth.feature

```diff
@@ -25,8 +25,201 @@ Feature: Connections - OAuth
       | Google Calendar | Test-Google-Calendar-connection |
       | Twitter         | Test-Twitter-connection         |
 
+  @oauth-gmail
+  Scenario: Testing Gmail OAuth connector
+    Given delete emails from "jbossqa.fuse@gmail.com" with subject "syndesis-tests"
+    When navigate to the "Settings" page
+    And fill all oauth settings
+    And navigate to the "Connections" page
+    And create connections using oauth
+      | Gmail | Test-Gmail |
+    And navigate to the "Home" page
+    And click on the "Create Integration" button to create a new integration
+    Then check visibility of visual integration editor
+
+    Then check that position of connection to fill is "Start"
+    When select the "Test-Gmail" connection
+    And select "Receive Email" integration action
+    And fill in values 
+      | labels | syndesis-test |
+    And click on the "Done" button
+
+    Then check that position of connection to fill is "Finish"
+    When select "Log" integration step  
+    And fill in values
+      | Message Body | checked |
+    And click on the "Done" button
+
+    And click on the "Save" button
+    And set integration name "OAuth-gmail-test"
+    And publish integration
+
+    When navigate to the "Integrations" page
+    Then Integration "OAuth-gmail-test" is present in integrations list
+    # wait for integration to get in active state
+    And wait until integration "OAuth-gmail-test" gets into "Running" state
+
+    #give gmail time to receive mail
+    When send an e-mail
+
+    When sleep for "6000" ms
+    Then validate that logs of integration "OAuth-gmail-test" contains string "syndesis-tests"
+    And delete emails from "jbossqa.fuse@gmail.com" with subject "syndesis-tests"
+  
+  @oauth-twitter
+  Scenario: Testing Twitter OAuth connector
+    Given clean all tweets in twitter_talky account
+    When navigate to the "Settings" page
+    And fill all oauth settings
+    And navigate to the "Connections" page
+    And create connections using oauth
+      | Twitter | Twitter-test |
+    And navigate to the "Home" page
+    And click on the "Create Integration" button to create a new integration
+    Then check visibility of visual integration editor
+    
+    Then check that position of connection to fill is "Start"
+    When select the "Twitter-test" connection
+    And select "Mention" integration action
+    #And click on the "Done" button
```

Comment:
@mmuzikar if you'll remember, remove this line in your next PR, I didn't want to bother you anymore because of single line :)